examples/todo-list/src/repositories/todo.repository.ts

```diff
@@ -36,23 +36,21 @@ export class TodoRepository extends DefaultCrudRepository<
       'todoList',
       todoListRepositoryGetter,
     );
+    this.registerInclusion('todoList', this.todoList.inclusionResolver);
   }
 
-  async find(
+  protected async includeRelatedModels(
+    entities: Todo[],
     filter?: Filter<Todo>,
-    options?: Options,
-  ): Promise<TodoWithRelations[]> {
-    // Prevent juggler for applying "include" filter
-    // Juggler is not aware of LB4 relations
-    const include = filter && filter.include;
-    filter = {...filter, include: undefined};
-
-    const result = await super.find(filter, options);
+    _options?: Options,
+  ): Promise<(Todo & TodoRelations)[]> {
+    const result = entities as (Todo & TodoRelations)[];
```

Comment:
Same here

Reply:
I am going to remove `includeRelatedModels`, it's no longer needed.

Reply:
Fixed in d0db330a6